azero/contracts/advisory/trait/lib.rs

```diff
@@ -0,0 +1,36 @@
+#![cfg_attr(not(feature = "std"), no_std, no_main)]
```

Comment:
Question: why do we need this trait thing? Isn't this for interacting with the contract from another contract?

Reply:
not strictly needed, maybe e2e test, but removing